vagrant/README.md

```diff
@@ -51,12 +65,34 @@ and after that it will install/upgrade collector in the vagrant environment.
 
 To remove collector please use:
 
+```bash
+sumo-make clean
+```
+
+or
+
 ```bash
 /sumologic/vagrant/Makefile clean
 ```
 
-List of other useful commands in the Makefile:
+List of other useful targets:
 
 - `expose-prometheus` - exposes prometheus on port 9090 of virtual machine
 - `expose-grafana` - exposes grafana on port 8080 of virtual machine
 - `apply-avalanche` - run one pod deployment of avalanche (metrics generator)
+
+## Test
+
+In order to quickly test whether sumo-kubernetes-collection works, one can use `receiver-mock` for that purpose.
+
+To check receiver-mock logs please use:
+
+```
+kubectl logs $(kubectl get pod -l app=receiver-mock -o jsonpath="{.items[0].metadata.name}"  -n receiver-mock) -n receiver-mock
+```
+
+To check metrics exposed by receiver-mock please use:
+
+```
+kubectl exec $(kubectl get pod -l app=receiver-mock -o jsonpath="{.items[0].metadata.name}"  -n receiver-mock) -it -n receiver-mock -- curl http://localhost:3000/metrics
+```
```

Comment:
We could add those commands as new targets to the `Makefile`. Not necessary in this PR :)

Reply:
> We could add those commands as new targets to the `Makefile`. Not necessary in this PR :)

I'll add this in a new PR